src/encoded/tests/conftest.py

```diff
@@ -199,3 +203,52 @@ def embed_testapp(app):
 @pytest.fixture
 def wsgi_app(wsgi_server):
     return webtest.TestApp(wsgi_server)
+
+
+def _check_server_is_up(output):
+    """ Polls the given output file to detect
+
+        :args output: file to which server is piping out
+        :returns: True if server is up, False if failed
+    """
+    tries = 5
+    while tries > 0:
+        output.seek(0)  # should be first thing to be output.
```

Comment:
Not sure I see what this is about, in spite of the comment that hints at effect but not mechanism. Is something clearing the output or resetting the position to the head of the output on each loop?